### PR TITLE
[VEN-2812]: Deployment files for Oracles on OP mainnet

### PR DESCRIPTION
**Pull request by Debugger022**
## Description

Resolves VEN-2812

Deployed addresses:
## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: helpers/deploymentConfig.ts [hunk 1]
```diff
@@ -1139,6 +1154,38 @@ export const assets: Assets = {
...
+      oracle: "chainlink",
+    },
+    {
+      token: "WETH",
+      address: "0x4200000000000000000000000000000000000006",
+      oracle: "chainlink",
+    },
+    {
+      token: "XVS",
+      address: "",
```
Comment (chechu) [inline helpers/deploymentConfig.ts:1185]:
```suggestion
      address: "0x4a971e87ad1F61f7f3081645f52a99277AE917cF",
```

Comment (Debugger022) [reply helpers/deploymentConfig.ts]: fixed, https://github.com/VenusProtocol/oracle/pull/225/commits/6a33f37cd25fdadc88e25e7d89d6debb947d043b

File: helpers/deploymentConfig.ts [hunk 2]
```diff
@@ -309,6 +317,13 @@ export const chainlinkFeed: Config = {
...
+  opmainnet: {
+    WBTC: "0xD702DD976Fb76Fffc2D3963D037dfDae5b04E593",
+    WETH: "0x13e3Ee699D1909E989722E753853AE30b17e08c5",
+    USDC: "0x16a9FA2FDa030272Ce99B29CF780dFA30361E0f3",
+    USDT: "0xECef79E109e997bCA29c1c0897ec9d7b03647F5E",
+    OP: "0x0D276FC14719f9292D5C1eA2198673d1f4269246",
+  },
 };
 
 export const redstoneFeed: Config = {
```
Comment (GitGuru7) [inline helpers/deploymentConfig.ts:329]: Shouldn't we add XVS redstone feed here?

Comment (chechu) [reply helpers/deploymentConfig.ts:329]: https://github.com/VenusProtocol/oracle/pull/225/commits/2e8b8692ea8a48d0f10c0ad70a92ae4a38be3726

**Review by GitGuru7 — APPROVED** [review]

**Review by kkirka — APPROVED** [review]

**Pull request merged**